### PR TITLE
Winter boots

### PR DESCRIPTION
**Pull request by Dhaeleena**
Adds Winter boots for every department! Adds said boots to the loadout. Adds boots to all the closets and wardroves that had winter jackets. Adds boots and jackets to some closets that had neither and fitted in.

http://i.imgur.com/GBM32RB.png
http://i.imgur.com/KQtRE6B.png
http://i.imgur.com/lCmGRtz.png
http://i.imgur.com/rgyvE17.png
http://i.imgur.com/aCsIcSB.png
http://i.imgur.com/2708nHO.png
http://i.imgur.com/2kmW3rG.png
http://i.imgur.com/vqDvB4a.png
http://i.imgur.com/zZi4iVv.png

### CODE REVIEW

File: code/modules/clothing/shoes/boots.dm [hunk 1]
```diff
@@ -47,6 +47,52 @@
...
+	icon_state = "winterboots_med"
+
+/obj/item/clothing/shoes/boots/winter/mining
+	name = "mining winter boots"
+	desc = "A pair of winter boots. These ones are lined with greyish fur, and their trim is golden!"
+	icon_state = "winterboots_mining"
+
+/obj/item/clothing/shoes/boots/winter/supply
+	name = "supply winter boots"
+	desc = "A pair of winter boots.These ones are lined with the galactic cargoria colors!"
```
Comment (Anewbe) [inline code/modules/clothing/shoes/boots.dm]: cargonia

Comment (Neerti) [inline code/modules/clothing/shoes/boots.dm]: There seems to be a missing space in the middle, as well.

File: code/modules/clothing/shoes/boots.dm [hunk 2]
```diff
@@ -47,6 +47,52 @@
...
+	icon_state = "winterboots_sup"
+
+/obj/item/clothing/shoes/boots/winter/hydro
+	name = "hydroponics winter boots"
+	desc = "A pair of winter boots.These ones are lined with brown fur, and their trim is ambrosia green"
```
Comment (Belsima) [inline code/modules/clothing/shoes/boots.dm]: Missing a space before 'these'.

File: code/modules/clothing/shoes/boots.dm [hunk 3]
```diff
@@ -47,6 +47,52 @@
...
+	icon_state = "winterboots_mining"
+
+/obj/item/clothing/shoes/boots/winter/supply
+	name = "supply winter boots"
+	desc = "A pair of winter boots. These ones are lined with the galactic cargonia colors!"
+	icon_state = "winterboots_sup"
+
+/obj/item/clothing/shoes/boots/winter/hydro
+	name = "hydroponics winter boots"
+	desc = "A pair of winter boots.These ones are lined with brown fur, and their trim is ambrosia green"
```
Comment (Belsima) [inline code/modules/clothing/shoes/boots.dm]: Still missing a space before 'these', by the way. Missed it in your last commit.

Comment (Dhaeleena) [inline code/modules/clothing/shoes/boots.dm]: That's a different kind of boots not the cargo boots, I was not aware the hydro boots missed the space

**Comment by Anewbe** [comment]
@Dhaeleena Any chance you're going to add that space?

**Comment by Dhaeleena** [comment]
Been a while away from the computer, i will do that as soon as i can yea

**Comment by SpadesNeil** [comment]
@Dhaeleena ya doof finish the thing.

**Comment by Anewbe** [comment]
Gonna merge this and fix the line, no big deal.

**Pull request merged**